common/src/mq/message-broker-channel.ts

```diff
@@ -75,9 +73,10 @@ export class MessageBrokerChannel {
                     stringPayload = '{}';
             }
 
-            const msg = await this.channel.request(eventType, StringCodec().encode(stringPayload), {
-                timeout: timeout || MQ_TIMEOUT,
-            });
+            // NOTE: If get NATS TIMEOUT error to quckly resolve just uncomment next line.
+            // And then, implement async processing of operation.
+            // const msg = await this.channel.request(eventType, StringCodec().encode(stringPayload), { timeout: 300000 });
+            const msg = await this.channel.request(eventType, StringCodec().encode(stringPayload));
```

Comment:
without timeout pass in the request function.  it will use 30 sec as default, when you publish a big policy with many schema then it will timeout.